packages/devtools-launchpad/src/components/Root.css

```diff
@@ -1,6 +1,18 @@
 :root.theme-light,
 :root .theme-light {
   --theme-search-overlays-semitransparent: rgba(221, 225, 228, 0.66);
+  --theme-codemirror-gutter-background: #f4f4f4;
+}
+
+:root.theme-dark,
+:root .theme-dark {
+  --theme-search-overlays-semitransparent: rgba(42, 46, 56, 0.66);
+  --theme-codemirror-gutter-background: #262b37;
+}
+
+:root.theme-firebug,
+:root .theme-firebug {
+  --theme-codemirror-gutter-background: #ebeced;
```

Comment:
I agree and mentioned it in the PR. I just wanted to do it in steps:
- cleanup devtools-core variables.css (this PR) 
- resynchronize variables.css from mc
- update debugger.html with a variables.css (or other name) that contains these 2 var, but renamed without --theme (which in devtools more or less means "comes from variables.css")
- remove the variables from here

I can just fold steps 1 and 4 into one and get rid of the values immediately in devtools-core. It's not like debugger.html will be impacted right away.